integration/commands_administration_test.go

```diff
@@ -1071,6 +1071,8 @@ func TestCommandsAdministrationServerStatusFreeMonitoring(t *testing.T) {
 }
 
 func TestCommandsAdministrationServerStatusStress(t *testing.T) {
+	// TODO rewrite using teststress.Stress
```

Comment:
Wow wow! :)

Reply:
wow? Ok, but that PR is neither approved nor rejected, and I'm not sure what to do with that comment

Reply:
No, no I'm still reading :) I like the idea, do we need an issue number for these new TODOs?